_projects/ft_workflow_project.md

```diff
@@ -149,11 +147,17 @@ Remember to use the `--file jlesc.bib` with the `cite` tag.
 
 {% bibliography --cited --file jlesc.bib %}
 
+## Person-Month efforts in 2016/2017
+
+{:.person-months-table.table.table-bordered.table-hover.table-sm}
+| {% person benoit_a %} | 1.0 PM |
+| {% person cappello_f %} | 1.0 PM |
+| {% person robert_y %} | 1.0 PM |
+| {% person cavelan_a %} | 3.0 PM |
 
```

Comment:
Please remove the PMs as well as the whole section here. We will follow a different, more private procedure this year. More on this later.